Documentation/Contributors/TestingGuide/README.md

```diff
@@ -502,6 +503,46 @@ it("can declare automatic uniforms", function () {
 });
 ```
 
+#### Debugging Rendering Tests
+
+Rendering tests typically render to a 1x1 pixel canvas. This is so each test runs as
+quickly as possible. However, when regressions happen, it is difficult to tell why the test is failing since the image is too small to see. To make debugging tests easier,
+`createScene()` has options to adjust the canvas size to give a better preview of what
+the camera sees.
+
+An example debug workflow might look like this:
+
+1. Use `fit()` to focus on the test that is failing.
+2. Use the `debugWidth` and `debugHeight` options for `createScene()` to make the canvas larger.
+3. Put a breakpoint in the code around where the first rendering code happens, such as a call of `scene.renderForSpecs()`.
+4. Step through the test. After each render, check the browser window to see the frame that was just rendered.
+
+```js
+beforeAll(function () {
+  scene = createScene({
+    // Create a 300x300 px canvas so we can see what the camera sees.
+    // This is intended for temporary debugging, do not commit these debug options.
```

Comment:
Would it be possible to send these options down with a command line argument instead of in the code? That way it wouldn't get accidentally committed.